docs/Compile/BuildLinux.md

```diff
@@ -212,13 +212,13 @@ Specify CMake parameters: -Dtigervnc_DIR=[TigerVNC installation path]/lib/cmake
   + WinPR_DIR: [freerdp installation path]/lib/cmake/WinPR2
   + FreeRDP_DIR: [freerdp installation path]/lib/cmake/FreeRDP2
   + FreeRDP-Client_DIR: [freerdp installation path]/lib/cmake/FreeRDP-Client2
-  + tigervnc_DIR: [TigerVNC installation path]/lib/cmake
+  + TigerVNC_DIR: [TigerVNC installation path]/lib/cmake
   + vncclient_DIR: [libvncserver installation path]/lib/cmake/LibVncServer
```

Comment:
Not sure "lib/cmake/LibVncServer" is correct

Reply:
I am will chanage to libVNCServer.